src/eltypes.jl

```diff
@@ -540,13 +555,13 @@ function UnionT(f::Meta.Field, convert)
     UT = UnionT{
         f.type.mode,
         typeids,
-        Tuple{(juliaeltype(x, buildmetadata(x), convert) for x in f.children)...},
+        Tuple{(juliaeltype(x, buildmetadata(x), convert, false) for x in f.children)...},
```

Comment:
Can you help me understand why we hard-code `false` in a number of places for the `allvalid` argument? vs. in other places calling `all(validity)`?